specification/network/resource-manager/Microsoft.Network/stable/2019-02-01/expressRouteGateway.json

```diff
@@ -595,19 +585,9 @@
             ],
             "properties": {
                 "provisioningState": {
-                    "type": "string",
                     "readOnly": true,
```

Comment:
The definition already contains "readOnly": true. This should be removed from here.